snapshot/src/test/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksumTest.java

```diff
@@ -123,8 +130,32 @@ public void shouldPersistChecksum() throws Exception {
     assertThat(actual.getCombinedValue()).isEqualTo(expectedChecksum.getCombinedValue());
   }
 
+  @DisabledIfEnvironmentVariable(named = "GITHUB_ACTIONS", matches = "true")
```

Comment:
:upside_down_face: I think `CI` is the canonical environment variable set by all CI systems: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Good to know, but I wonder why the need for both :smile: